code/modules/mob/language/station.dm

```diff
@@ -289,7 +289,9 @@
 
 /datum/language/bug/check_special_condition(var/mob/other)
 	if(istype(other, /mob/living/silicon))
-		return 1
+		var/mob/living/silicon/S = other
+		if(S.can_hear_hivenet)
+			return 1
```

Comment:
```suggestion
			return TRUE
```

Reply:
Nothing else in this section of code uses TRUE/FALSE. I'll do it anyway, but it'll look weird I think.